api/v1beta1/cinderbackup_types.go

```diff
@@ -72,6 +72,11 @@ type CinderBackupSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	// TLS - Parameters related to the TLS
 	TLS tls.Ca `json:"tls,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=memcached
+	// Memcached instance name.
+	MemcachedInstance string `json:"memcachedInstance"`
```

Comment:
same concern here, a new operator update will update the Spec with this parameter and trigger an unwanted restart for cinder backup. We should turn it into a *string to make sure we can omit it everywhere and be fully backward compatible,